pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -520,13 +520,22 @@ void registerBackendAgnosticAlgorithms(py::module_ &m) {
                       const std::vector<std::vector<size_t>> &,
                       const std::vector<bool> &,
                       const std::vector<std::vector<ComplexT>> &>())
+        .def(py::init<const std::vector<std::string> &,
+                      const std::vector<std::vector<ParamT>> &,
+                      const std::vector<std::vector<size_t>> &,
+                      const std::vector<bool> &,
+                      const std::vector<std::vector<ComplexT>> &,
+                      const std::vector<std::vector<size_t>> &>())
```

Comment:
Nothing for now, but we should probably add docs for these to be visible in Python bindings later -- I imagine it may get tricky to understand what carries across to what.